Troupon.Catalog.Core.Domain/Dtos/DealDetailsDto.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Troupon.Catalog.Core.Domain.Entities.Common;
+using Troupon.Catalog.Core.Domain.Enums;
+
+namespace Troupon.Catalog.Core.Domain.Dtos
```

Comment:
You see, link to my comment on the namespaces for DTO in your "troupon.checkout" PR, here the DTO is in Domain not in Applicaiton.

I will add a task to TFS so we can address this next sprint maybe (or if we have time this sprint)

Reply:
Great idea! Please map this task to TFS so we can work on it later

Reply:
I map it here => https://tfs18.utcapp.com/CustomerService/ApplicationReferecenArchitectureInitiave/_workitems?id=46944&_a=edit